docs/user/ppl/cmd/stats.rst

```diff
@@ -238,6 +238,27 @@ Example::
     | 2.8613807855648994 |
     +--------------------+
 
+TAKE
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: TAKE(field [, size]). Return original values of a field.
```

Comment:
could you add more detail? e.g. how does the original values picked, random? or ordered?

Reply:
I'll add `Take command does not guarantee on order of values.`, let me know if i should mention top_hits here.

if it's pushed down it's top_hits order, in memory it is same as `head`